models/mailchimp/mailchimp_sent_to.sql

```diff
@@ -0,0 +1,7 @@
+select distinct
+	list_id, campaign_id, email_id, email_address, status, send_time as sent_date
+from demo_data.mailchimp_sent_to a
```

Comment:
The first tier of any set of models needs to be 1:1 to the raw data tables with no transformation applied; this is how we're standardizing the mapping layer. It's an important component of how the package management function of dbt is getting designed.

`mailchimp_sent_to` needs a mapper before it gets transformed.